ui/job-view/headerbars/TierIndicator.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ButtonGroup, Button } from 'reactstrap';
+
+const TIERS = ['1', '2', '3'];
+
+export default function TierIndicator(props) {
+  const { filterModel } = props;
+  const shownTiers = filterModel.urlParams.tier || [];
+
+  return (
+    <ButtonGroup>
+      {TIERS.map(tier => {
+        const isOnlyTier = shownTiers.length === 1 && tier === shownTiers[0];
+        return (
+          <Button
+            onClick={() => filterModel.toggleFilter('tier', tier)}
+            title={
+              isOnlyTier
+                ? "Can't Toggle because least one tier must be selected at all times"
+                : 'Toggle tier '.concat(tier).concat(' jobs')
```

Comment:
Rather than concatenation, we use the syntax of:

    `Toggle tier ${tier} jobs`

Note he back-ticks rather than quotes.